consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
 
 
+class Consultation(models.Model):
+    # May need something better long-term
+    class Organisation(models.TextChoices):
```

Comment:
Do we need this yet? Maybe we should leave out til we know more

Reply:
Now removed.